binder/binder.c

```diff
@@ -2435,7 +2435,11 @@ static int binder_translate_binder(struct flat_binder_object *fp,
 		ret = -EINVAL;
 		goto done;
 	}
+#if (LINUX_VERSION_CODE >= KERNEL_VERSION(5,15,2))
+	if (security_binder_transfer_binder(proc->tsk->real_cred, target_proc->tsk->real_cred)) {
```

Comment:
Looking at the most recent version of binder_translate_binder() in the kernel (see [this link](https://elixir.bootlin.com/linux/v5.15.2/source/drivers/android/binder.c#L2034)), the function is invoked slightly differently:
`if (security_binder_transfer_binder(proc->cred, target_proc->cred)) {`
This differs from your code. It is possible that your code works nonetheless, but I would like to keep binder.c in the modules here as consistent as possible to the version in the upstream kernel. So, could you please change this line accordingly? Please also check the "security_binder_*" calls that you changed below to make sure that the code matches as much as possible to "binder.c" in the kernel.This makes the modules here much easier to maintain.

Reply:
Right. Updating as soon as I can

Reply:
I changed to be in same style as Android binder.c, and I noticed there's more changes in the binder.c and struct binder_proc (new fields). A new PR may be needed